acceptance/internal/config.go

```diff
@@ -64,6 +65,8 @@ type TestConfig struct {
 	Ignore []string
 
 	CompiledIgnoreObject *ignore.GitIgnore
+
+	EnvMatrix map[string][]string
```

Comment:
Please also document the behaviour of this configuration field here. Ideally, anyone can read the `TestConfig` struct to figure out the options supported and how they work.